about.html

```diff
@@ -0,0 +1,120 @@
+                    <ul>
+                        <li>
+                            <a href="index.html">Home</a>
+                        </li>
+                        <li>
+                            <a href="about.html" class="active-tab">About</a>
+                        </li>
+                        <li>
+                            <a href="index.html#main-program">Program</a>
+                        </li>
+                        <li>
+                            <a href="https://apply.microverse.org/" target="_blank" rel="noopener">Join</a>
+                        </li>
+                    </ul>
+            </nav>
+        </header>
+
+        <section class="introduction">
+            <h1 class="introduction__header">
+                Microverse Full-Stack web development program
+            </h1>
+            <div class="introduction__description-container">
+                <p class="introduction__description">
+                    At Microverse, students learn through a very unique approach by pair programming and collaborating with other students in real-time, just as they would with colleagues of a distributed team in a real company.
+                    Students learn multiple programming languages and Computer Science fundamentals, through working on real projects with other students around the world.
+                </p>
+            </div>
+            <p class="contact-us-email">
+                Please contact us per Email for any further questions about Microverse program
+                <a class="contact-us-link" href="mailto:admissions@microverse.org">admissions@microverse.org</a>
+            </p>
+        </section>
+
+        <section class="alumni-opinion">
+            <h3>
+                What do alumni think of Microverse?
+            </h3>
+            <p class="alumni-description">
+                Microverse accepted its first cohort in early 2018 and its first graduates started receiving job offers in September 2018. Here are some of their comments, reviews, and videos. 
+            </p>
+
+            <div class="alumni-description-links">
+                <div class="img-container">
+                    <img src="images/canda.png" alt="Canada flag">
+                    <div class="img-overlay">
+                        <a href="https://support.microverse.org/en/articles/2590022-what-do-alumni-think-of-microverse/#paul-rail-toronto-canada">
+                            Paul Rail (Toronto, Canada)
+                        </a>
+                    </div>
+                    </div>
+                <div class="img-container">
+                    <img src="./images/serbia.png" alt="Serbia flag">
+                    <div class="img-overlay">
+                        <a href="https://support.microverse.org/en/articles/2590022-what-do-alumni-think-of-microverse/#sava-vuckovic-nis-serbia">
+                        Sava Vuckovic (Nis, Serbia)
+                        </a>
+                    </div>
```

Comment:
- [ ] Due to the insistence of this project to use the colors provided, please change this blue color to the orange one in the design to comply with the project requirements.